### PR TITLE
NAS-109686 / 21.04 / "show_if" should work on all types

### PR DESCRIPTION
**Pull request by AlexKarpov98**

### CODE REVIEW

**Review by Ornias1993 — APPROVED** [review]
Looks like a clean and simple fix :)

(and thanks for the super fast bugfix @AlexKarpov98 )

File: src/app/pages/common/entity/utils.ts [hunk 1]
```diff
@@ -433,6 +433,10 @@ export class EntityUtils {
     if (fieldConfig) {
 
       if (fieldConfig['type']) {
+        if (schemaConfig.schema.show_if) {
+          fieldConfig['relation'] = this.createRelations(schemaConfig.schema.show_if, parentName);;
```
Comment (sonicaj) [inline src/app/pages/common/entity/utils.ts:437]: Multiple `;;` ?

**Review by sonicaj — APPROVED** [review]

File: src/app/pages/common/entity/utils.ts (hunk 1 above)
Comment (AlexKarpov98) [reply src/app/pages/common/entity/utils.ts]: Fixed

**Review by ryousaf — APPROVED** [review]
LGTM

**Pull request merged**